gneiss/_model.py

```diff
@@ -40,43 +38,9 @@ def summary(self):
         """ Print summary results """
         pass
 
-    @classmethod
-    def read_pickle(self, filename):
-        """ Reads Model object from pickle file.
-
-        Parameters
-        ----------
-        filename : str or filehandle
-            Input file to unpickle.
-
-        Returns
-        -------
-        Model
-
-        Notes
-        -----
-        Warning: Loading pickled data received from untrusted
-        sources can be unsafe. See: https://wiki.python.org/moin/UsingPickle
-        """
-
-        if isinstance(filename, str):
-            with open(filename, 'rb') as fh:
-                res = pickle.load(fh)
-        else:
-            res = pickle.load(filename)
-
-        return res
-
-    def write_pickle(self, filename):
-        """ Writes Model object to pickle file.
-
-        Parameters
-        ----------
-        filename : str or filehandle
-            Output file to store pickled object.
-        """
-        if isinstance(filename, str):
-            with open(filename, 'wb') as fh:
-                pickle.dump(self, fh)
-        else:
-            pickle.dump(self, filename)
+    def percent_explained(self):
+        """ Proportion explained by each principal balance."""
+        # Using sum of squares error calculation (df=1)
```

Comment:
Sure thing!

The degrees of freedom boils down to the nature of [variance](https://en.wikipedia.org/wiki/Variance#Population_variance_and_sample_variance).
When calculating population variance, you are just doing

`sigma = (1/n)* sum([(y[i] - np.mean(y))**2 for i in range(0, n)])`

The problem is, this is actually a biased estimator for calculating variance - it is off by 1 degree of freedom.  The way to obtain an unbiased estimator of variance is

`sigma = (1 / (n+1))* sum([(y[i] - np.mean(y))**2 for i in range(0, n)])`

so by specifying `ddof=1`, I am forcing numpy to calculate the sample variance - because by default, numpy is calculating the biased estimate of variance.  Super subtle, but super important.